sql/core/src/test/scala/org/apache/spark/sql/CachedTableSuite.scala

```diff
@@ -670,4 +677,139 @@ class CachedTableSuite extends QueryTest with SQLTestUtils with SharedSQLContext
       assert(spark.read.parquet(path).filter($"id" > 4).count() == 15)
     }
   }
+
+  test("SPARK-19993 simple subquery caching") {
+    withTempView("t1", "t2") {
+      Seq(1).toDF("c1").createOrReplaceTempView("t1")
+      Seq(1).toDF("c1").createOrReplaceTempView("t2")
```

Comment:
@cloud-fan sorry... actually i had some of these tests combined and when i split, i forgot to remove this. Will fix it.